src/map.h

```diff
@@ -2352,6 +2352,10 @@ class map
 #if defined(TILES)
         bool draw_points_cache_dirty = true;
         std::map<int, std::map<int, std::vector<tile_render_info>>> draw_points_cache;
+        int prev_min_col;
+        int prev_max_col;
+        int prev_min_row;
+        int prev_max_row;
```

Comment:
Where are these four variables assigned?

Reply:
Oh, I forgot to set them to the current values, my bad. Coincidentally this still "solves" the issue, but it does so by effectively disabling the draw cache entirely, which wasn't my intention. Whoops.